docs/presentation/content/CalculatedPropertiesSpecification.md

```diff
@@ -47,6 +48,22 @@ and [Ruleset Variables](../advanced/ECExpressions.md#ruleset-variables-user-sett
 
 ![Example of using "value" attribute](./media/calculatedpropertiesspecification-with-value-attribute.png)
 
+### Attribute: `type`
+
+Specifies return type of the calculated property. If a type that is not one of `"string" | "int" | "long" | "dateTime" | "boolean" | "bool" | "double"` is provided, calculated property won't appear in the property pane. If evaluated expression can't be converted to the specified type an error will be thrown in the frontend code.
+
+|                   |                                              |
+| ----------------- | -------------------------------------------- |
+| **Type**          | `"string" \| "int" \| "long" \| "dateTime" \| "boolean" \| "bool" \| "double"` |
```

Comment:
This should match the type in Attributes table above

Reply:
Adjusted the type in the table above